src/index.js

```diff
@@ -63,8 +65,12 @@ FlowWebpackPlugin.prototype.apply = function(compiler) {
         }
     })
 
-    compiler.plugin('run', runCallback(plugin.options.failOnError))
-    compiler.plugin('watch-run', runCallback(plugin.options.failOnErrorWatch))
+    if(plugin.options.bundle) {
+        compiler.plugin('done', runCallback())
```

Comment:
I don't find nice to omit an argument of `runCallback()` when the definition expects one. Even though it a valid in JS. Passing `false` would be easier to read.